web/components/feed/contract-activity.tsx

```diff
@@ -28,38 +28,40 @@ export function ContractActivity(props: {
   className?: string
   betRowClassName?: string
 }) {
-  const { user, mode, contractPath, className, betRowClassName } = props
+  const { user, mode, contractPath, className, betRowClassName, contract } =
+    props
 
-  const contract = useContract(props.contract.id) ?? props.contract
+  const liveContract = useContractWithPreload(contract) ?? props.contract
```

Comment:
I would just leave the name as `contract`. Many of our hooks return live updating data, but we don't prefix the name.